### ISSUE TITLE
lock file

### ISSUE THREAD

**Issue by cg-** [cerebrate/wabash#3]
I'm having lots of issues with the lockfile not successfully being deleted when shutting down the computer, causing the next autostart to not occur successfully. I have to go in manually and delete the lockfile, then the program successfully starts up.

**Comment by cerebrate** [comment]
Yeah, it's a known bug, unfortunately: so far, I've not found a way to guarantee that processes will shut down in the right order to ensure that the lock file will be cleaned up. I'm working on it, but I don't have an ETF yet.

**Comment by cerebrate** [comment]
@cg- This may be fixed by the next revision of wabashd which includes a precheck to ensure the lockfile is valid in lieu of being able to guarantee deletion on shutdown. It's in the wsl-translinux repository (per new installation instructions in the README) now; grab it and see if it works for you?

**Comment by cg-** [comment]
Works great! Thanks!

For anyone who might not have that version, my solution was to not have wabash.exe start automatically. Instead, I created a script that removed the lockfile called /usr/local/sbin/clear-wabash-lock.sh. Then I created a task scheduler task that ran bash.exe -c /usr/local/sbin/clear-wabash-lock.sh at login, and afterwards launched wabash.exe.

This updated version of wabashd seems to resolve the issue though, running at start through the right-click menu on the taskbar icon works fine.

Thanks so much, this program is super useful. Nice to be able to backup my Windows machine with borg using cron :)
